src/pages/public_registration/ActionPanelContent.tsx

```diff
@@ -27,6 +33,8 @@ export const ActionPanelContent = ({
 }: ActionPanelContentProps) => {
   const { t } = useTranslation();
   const dispatch = useDispatch();
+  const history = useHistory();
+  const currentPath = history.location.pathname;
```

Comment:
Om vi bare trenger `location` kan vi bruke `useLocation` i stedet, for å kanskje slippe unna noen rerenders.
Kan da bruke `location.pathname.startsWith(...)` lenger ned her
```suggestion
  const location = useLocation();
```

Reply:
Trenger history for å redirigere lenger ned, så denne endringen utgår